### PR TITLE
Continuing of the extending on the NBI

### PR DESCRIPTION
**Pull request by SeanCondon**
> I think I merged the previous PR by accident - this might have broken the integration tests

The think to look at here is the diags.proto and the new `ChangeService` with just 2 methods:

- `ListNetworkChanges` and 
- `ListDeviceChanges`

I've also added 2 CLI commands

- `onos config watch device-changes <device-change-id wildcard>` and
- `onos config watch network-changes <network-change-id wildcard>`

I haven't finished wiring these up to the actual stores yet.

There are test cases for the CLI commands

### CODE REVIEW

File: pkg/cli/root.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func GetCommand() *cobra.Command {
 
 	cmd.AddCommand(getGetCommand())
 	cmd.AddCommand(getAddCommand())
-	//cmd.AddCommand(getRemoveCommand())
```
Comment (Andrea-Campanella) [inline pkg/cli/root.go]: why is this removed ?

File: pkg/northbound/diags/diags.proto [hunk 2]
```diff
@@ -123,6 +103,10 @@ message ListDeviceChangeRequest {
     // subscribe indicates whether to subscribe to events (e.g. ADD, UPDATE, and REMOVE) that occur
     // after all devices have been streamed to the client
     bool subscribe = 1;
+
+    // option to specify a specific device change - if blank or '*' then select all
+    // Can support `*` (match many chars) or '?' (match one char) as wildcard
+    string changeid = 2;
```
Comment (kuujo) [inline pkg/northbound/diags/diags.proto]: FYI you can use gogo to generate a Go struct field that's properly cased:
```go
string change_id = 2 [(gogoproto.customname) = "ChangeID"];
```
and also properly typed
```go
string change_id = 2 [(gogoproto.customname) = "ChangeID", casttype = "github.com/onosproject/onos-config/pkg/types/change/network.ID"];
```

File: pkg/cli/root.go (hunk 1 above)
Comment (SeanCondon) [reply pkg/cli/root.go]: it had been commented out because you could not remove (unload) a plugin. I was removing the commented text here

File: pkg/northbound/diags/diags.proto (hunk 2 above)
Comment (SeanCondon) [reply pkg/northbound/diags/diags.proto]: Thanks  - will do that

File: pkg/cli/devicechanges.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
+// Copyright 2019-present Open Networking Foundation.
```
Comment (Andrea-Campanella) [inline pkg/cli/devicechanges.go]: maybe `watchdevicechanges.go` is a better name ?

File: pkg/cli/networkchanges.go [hunk 4]
```diff
@@ -0,0 +1,97 @@
+// Copyright 2019-present Open Networking Foundation.
```
Comment (Andrea-Campanella) [inline pkg/cli/networkchanges.go]: same maybe `watchnetworkchanges.go`
